packages/obonode/obojobo-chunks-multiple-choice-assessment/MCFeedback/schema.js

```diff
@@ -9,33 +9,33 @@ const TEXT_NODE = 'ObojoboDraft.Chunks.Text'
 const schema = {
 	blocks: {
 		'ObojoboDraft.Chunks.MCAssessment.MCFeedback': {
-			nodes: [{ match: [{ type: 'oboeditor.component' }], min: 1 }],
+			nodes: [{ match: [
+						{ type: 'ObojoboDraft.Chunks.ActionButton' },
+						{ type: 'ObojoboDraft.Chunks.Break' },
+						{ type: 'ObojoboDraft.Chunks.Code' },
+						{ type: 'ObojoboDraft.Chunks.Figure' },
+						{ type: 'ObojoboDraft.Chunks.Heading' },
+						{ type: 'ObojoboDraft.Chunks.HTML' },
+						{ type: 'ObojoboDraft.Chunks.IFrame' },
+						{ type: 'ObojoboDraft.Chunks.List' },
+						{ type: 'ObojoboDraft.Chunks.MathEquation' },
+						{ type: 'ObojoboDraft.Chunks.Table' },
+						{ type: 'ObojoboDraft.Chunks.Text' },
+						{ type: 'ObojoboDraft.Chunks.YouTube' }], min: 1 }],
```

Comment:
Yea, these present a bit of an issue for modularized chunks as well.  It would be nice to be able to ask Common.Registry for a list of nodes that can act as content types.

Reply:
Each `editor-registration.js` would have to have a a property like `isUsableAsContent`.  They register themselves as being able to be used and then the schema.js files just ask Common.Registry for a list of the nodes that have `isUsableAsContent === true`